src/js/background.js

```diff
@@ -497,7 +474,10 @@ Badger.prototype = {
       chrome.browserAction.setBadgeText({tabId: tabId, text: ""});
       return;
     }
-    var numBlocked = this.blockedTrackerCount(tabId);
+
+    let thisTab = this.tabData[tabId];
+    let numBlocked = thisTab ? thisTab.blockedCount : 0;
```

Comment:
I think we should just do `let nBlocked = this.tabData[tabId].blockedCount`, otherwise we are drawing a green zero for pages where PB is disabled or not active (like chrome pages).